roxctl/deployment/check/check.go

```diff
@@ -157,6 +159,10 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 	// d.firstFile needs to be populated before the printer is created
 	d.firstFile = d.files[0]
 
+	if d.namespace == "default" {
+		d.env.Logger().WarnfLn("Warning: The namespace defaulted to \"default\", if you want to check deployments in a custom namespace set it with '--namespace [NAMESPACE]' or '-n [NAMESPACE]'\n")
+	}
+
```

Comment:
Wouldn't this fit better to `Validate()`?

Reply:
Reasonable, I'm moving it.